generate_params_from_workflow.py

```diff
@@ -0,0 +1,121 @@
+#!/usr/bin/env python
+
+"""generate_params_from_workflow
+
+This script generate json parameter file from galaxy workflow using config_credentials.yml file provided.
+provisioned locally in a directory. This scripts connects to galaxy instance and grabs parameters 
+using bioblend galaxy instance.
+
+running syntax
+
+python generate_params_from_workflow.py \
+        -C galaxy_credentials.yml.sample \
+        -G 'ebi_cluster' \
+        -o $PWD \
+        -N 'scanpy_param_test' \
+        -W scanpy_clustering_workflow.json 
+
+File galaxy_credentials.yml.sample must contain url and key.
+"""
+
+import json
+import argparse
+import os.path
+from bioblend.galaxy import GalaxyInstance
+import yaml
+import logging
+from sys import exit
+
+
+def get_args():
+    arg_parser = argparse.ArgumentParser()
+    arg_parser.add_argument('-C', '--conf',
+                            required=True,
+                            help='A yaml file describing the galaxy credentials')
+    arg_parser.add_argument('-G', '--galaxy-instance',
+                            default='ebi_cluster',
+                            help='Galaxy server instance name')
+    arg_parser.add_argument('-o', '--output-dir',
+                            default="./",
+                            help='Path to output directory')
+    arg_parser.add_argument('-N', '--name-parameter',
+                            default='scanpy_params',
+                            required=True,
+                            help='Name of the parameter file to create')
```

Comment:
Is the script outputting something else in the outdir besides the parameters as json/yaml? otherwise maybe we can simply have an output flag that simply points to the produced files.

Reply:
Merging these two parameters I mean.